packages/inventory-vulnerabilities/src/Components/PaginationWrapper.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable camelcase */
+
+import { Pagination } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React, { Component } from 'react';
+
+class PaginationWrapper extends Component {
+    state = {};
+
+    shouldComponentUpdate(nextProps) {
+        if (nextProps.meta === this.props.meta) {
+            return false;
+        }
+
+        return true;
+    }
+    changePage = (_event, pageNumber) => this.setState({ ...this.state, page: pageNumber }, this.apply);
+
+    setPageSize = (_event, perPage) => this.setState({ ...this.state, page_size: perPage, page: 1 }, this.apply);
+
+    apply = () => {
+        this.props.apply(this.state);
```

Comment:
Same as above, this is really cryptic and hard to understand from parent component. If you want to send some stuff from child component to parent use callback functions instead of passing state to parent.